migrator.go

```diff
@@ -586,3 +608,67 @@ func (m Migrator) getColumnSequenceName(tx *gorm.DB, stmt *gorm.Statement, field
 	)
 	return
 }
+
+func (m Migrator) GetIndexes(value interface{}) ([]gorm.Index, error) {
+	indexes := make([]gorm.Index, 0)
+
+	err := m.RunWithValue(value, func(stmt *gorm.Statement) error {
+		result := make([]*Index, 0)
+		scanErr := m.DB.Raw(indexSql, m).Scan(&result).Error
+		if scanErr != nil {
+			return scanErr
+		}
+
+		if len(result) == 0 {
+			return nil
+		}
+
+		indexMap := groupByIndexName(result)
+
+		for _, idx := range indexMap {
+			if len(idx) == 0 {
```

Comment:
Is it necessary here? Assuming that there will be no empty records in the query, it will not go here, assuming that there will be empty records, it will be removed by groupByIndexName

Reply:
I so sorry, it seems necessary, because of `columnIndexMap := make(map[string][]*Index, len(indexList))`